ui/tests/acceptance/secrets/backend/kv/secret-test.js

```diff
@@ -577,7 +587,6 @@ module('Acceptance | secrets/secret/create', function(hooks) {
 
     await editPage.visitEdit({ backend, id: 'secret' });
     assert.notOk(editPage.hasMetadataFields, 'hides the metadata form');
-    assert.ok(editPage.showsNoCASWarning, 'shows no CAS write warning');
```

Comment:
confirmed with design no longer have these alerts. For a variety of reasons, but mostly because they are redundant and we are now more clear about permissions based on the view breakup.